ynr/apps/people/management/commands/people_clean_person_names.py

```diff
@@ -0,0 +1,35 @@
+from candidatebot.helpers import CandidateBot
+from django.core.management.base import BaseCommand
+from people.models import Person
+from popolo.models import OtherName
+
+
+class Command(BaseCommand):
+    help = "Replaces chars in person names that we don't want to be there."
+
+    def handle(self, *args, **options):
+        chars_to_replace = {"\xa0": " ", "\u200e": ""}
+
+        for char, replace in chars_to_replace.items():
+            qs = Person.objects.filter(name__contains=char).only("pk", "name")
+            for person in qs:
+                for other_name in person.other_names.all():
+                    other_name.name = other_name.name.replace(char, replace)
```

Comment:
A straight replace of `\xa0` with " " will edit "John\xa0Smith" to "John Smith" but presumably if we start off with "John\xa0 Smith" we get "John  Smith". Do we also want some kind of extraneous whitespace trimming here to remove duplicate spaces after replacement?